cmd/pg-schema-diff/plan_cmd.go

```diff
@@ -591,3 +597,19 @@ func planToJsonS(plan diff.Plan) string {
 	}
 	return string(jsonData)
 }
+
+func planToSqlS(plan diff.Plan) string {
+	if len(plan.Statements) == 0 {
+		return ""
+	}
+
+	var stmtStrs []string
+	for _, stmt := range plan.Statements {
+		ddl := strings.TrimSpace(stmt.DDL)
+		if !strings.HasSuffix(ddl, ";") {
```

Comment:
Is this not consistent across the outputted SQL? I would prefer to ensure are output is consistent inside the diff package than handling normalization in the CLI.